test/commands.js

```diff
@@ -64,7 +70,7 @@ let runBuyTokensCommand = async (command, state) => {
     state.purchases = _.concat(state.purchases,
       {tokens: tokens, rate: rate, wei: weiCost, beneficiary: command.beneficiary, account: command.account}
     );
-    state.balances[command.beneficiary] = (state.balances[command.beneficiary] || 0) + tokens;
+    state.balances[command.beneficiary] = (state.balances[command.beneficiary] || 0) + help.lif2LifWei(tokens);
```

Comment:
this is not a BigNumeber? if so `plus` ca be used instead of +

Reply:
good catch, I'll fix it

Reply:
Fixed in ee5f2e6